### PR TITLE
Added require-js url args to avoid wrong cache hits

### LINKED ISSUE #1877: Add cache busting for Sulu translations
At the moment the browser cache must be cleared after editing translations which are exported to web/admin/translations/sulu.xx.json.

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #1877 
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR introduces require.js url args to avoid wrong cache hits.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Gruntfile.js [hunk 1]
```diff
@@ -137,10 +153,16 @@ module.exports = function(grunt) {
...
-                    },
-                    prefix: ''
+                    patterns: [
+                        {
+                            match: /\(new Date\)\.getTime\(\)/,
+                            replacement: time.toString()
+                        },
+                        {
+                            match: /\/js\/main\.js/,
+                            replacement: '/dist/main.js'
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Gruntfile.js]: Any reason you are sometimes escaping the characters and sometimes are using quotes?

Comment (wachterjohannes) [inline src/Sulu/Bundle/AdminBundle/Gruntfile.js]: match is a regex there it has to be escape. the replacement is a simple string.

**Comment by danrot** [comment]
Setting breakpoints in develop mode should still work.

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**